mautic/mautic
# [UI] Remove borders from remaining tables

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR removes borders and stripes from tables, similar to the other PR regarding this topic, but now standardizing the entire interface.

Before:
![image](https://github.com/mautic/mautic/assets/116097999/191c65eb-01f4-4c42-a69e-3970ac89aa35)


After:

![image](https://github.com/mautic/mautic/assets/116097999/c88ac6a5-53bf-459a-8be9-2eb897176c09)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. API Credentials > List 
3. Assets > Create new > Save & Close > Details
4. Channels > Dynamic Content > New > Save & Close > Details
5. Campaigns > New > Save & Close > Details
6. Emails > New > Save & Close > Details
7. Forms > New > Save & Close > Details
8. Companies > New > Save & Close > Details
9. Contacts > Import > Select a file and import > Import history > Click on the import > Details
10. Contacts > New > Save & Close > Details
11. Segments > New > Save & Close > Details
12. Marketplace > Any plugin > See those 2 tables
13. Plugins > OneSignal > Fill all fields with anything > Save & Close > Channels > Mobile Notif > New > Save & Close > Details
14. Channels > Web Notif > New > Save & Close > Details
15. Components > Landing pages > New > Save & Close > Details
16. Reports > Open any > Details
17. Plugins > Twilio > Fill anything and set to Active > Save & Close > Channels > Text Messages > New > Save & Close > Details
18. Channels > Focus > New > Builder > Select any style and type > Save & Close > Details
19. Plugins > Twitter > Fill anything and set to Active > Save & Close > Channels > Social Monitoring > New > Save & Close > Details
20. Channels > Mobile Notif > See the list
21. Components > Forms > Click to see a form result (you need to have one created) > See the list

There are many other tables, but they're all the same situation as the above.
